### PR TITLE
[debounce] Add definition

### PR DESCRIPTION
**Pull request by Brianzchen**
- Links to documentation: https://github.com/component/debounce/blob/master/index.js#L10
- Link to GitHub or NPM: https://www.npmjs.com/package/debounce
- Type of contribution: new definition

Other notes:

### CODE REVIEW

**Comment by pascalduez** [comment]
**Suggestion**: Add [ES modules compat](https://github.com/component/debounce/blob/133fffd56187d56782ae4c898bc13aa75a18716c/index.js#L68)?
`import { debounce } from "debounce";`

**Comment by Brianzchen** [comment]
> **Suggestion**: Add [ES modules compat](https://github.com/component/debounce/blob/133fffd56187d56782ae4c898bc13aa75a18716c/index.js#L68)?
> `import { debounce } from "debounce";`

Wow just tested, that is so strange if you do any of these it returns the same obj. I'll make a fix
```
import mainDebounce, { debounce } from 'debounce';

console.log(mainDebounce);
console.log(mainDebounce.debounce);
console.log(debounce)
```

**Review by pascalduez — DISMISSED** [review]

File: definitions/npm/debounce_v1.x.x/flow_v0.83.x-/debounce_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+declare module 'debounce' {
+  declare type Cancelable = {
+    cancel: () => void,
+    flush: () => void
+  };
```
Comment (pascalduez) [inline definitions/npm/debounce_v1.x.x/flow_v0.83.x-/debounce_v1.x.x.js:5]: Exact or inexact ;)

**Comment by Brianzchen** [comment]
:/ The build really should have picked up on this.. Maybe something that was happening in travis doesn't seem to work in github actions anymore? Who's most familiar with the test runner that could take a look?

**Comment by pascalduez** [comment]
> The build really should have picked up on this

Yes, definitely. At least it works for me while testing locally.

**Review by pascalduez — APPROVED** [review]

**Pull request merged**